testing/unicode-character-database/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+pkgname=unicode-character-database
+pkgver=11.0.0
+pkgrel=0
+pkgdesc="Unicode Character Database"
+arch="noarch"
+license="BSD"
+url="http://www.unicode.org/"
+makedepends="libarchive-tools"
+source="UCD-$pkgver.zip::http://www.unicode.org/Public/zipped/$pkgver/UCD.zip
+	Unihan-$pkgver.zip::http://www.unicode.org/Public/zipped/$pkgver/Unihan.zip
+	LICENSE"
```

Comment:
I don't think we should provide our own license file.

Reply:
Any reason why not? It's the proper license according to https://www.unicode.org/copyright.html, which says that license applies to everything under http://www.unicode.org/Public

The package .zip doesn't have a license file itself, so I thought this was the proper way to do it. I could just keep it out and not install it if that is what you mean.

Reply:
IMHO, The proper way to do it would be report it upstream and ask them to ship the license file.

I think its better to keep it out and not install it.